src/main/java/com/revature/_1811_nov27_wvu/icebox/dao/UserHibernate.java

```diff
@@ -0,0 +1,72 @@
+package com.revature._1811_nov27_wvu.icebox.dao;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.hibernate.Transaction;
+
+import org.apache.log4j.Logger;
+import org.hibernate.Session;
+import org.hibernate.query.Query;
+import org.springframework.beans.factory.annotation.Autowired;
+
+
+import com.revature._1811_nov27_wvu.icebox.entity.User;
+
+public class UserHibernate implements UserDao{
+	private Logger log = Logger.getLogger(UserHibernate.class);
+	@Autowired
+	private static SessionFactory sf = new SessionFactory();
```

Comment:
SessionFactory should not be instantiated using the `new` keyword when the annotation `@Autowired` is used.